apps/meteor/app/api/server/lib/rooms.ts

```diff
@@ -100,10 +117,21 @@ export async function findAdminRoomsAutocomplete({ uid, selector }) {
 	};
 }
 
-export async function findChannelAndPrivateAutocompleteWithPagination({ uid, selector, pagination: { offset, count, sort } }) {
+export async function findChannelAndPrivateAutocompleteWithPagination({
+	uid,
+	selector,
+	pagination: { offset, count, sort },
+}: {
+	uid: string;
+	selector: { name: string };
+	pagination: { offset: number; count: number; sort: [string, number][] };
+}): Promise<{
+	items: IRoom[];
+	total: number;
+}> {
 	const userRoomsIds = Subscriptions.cachedFindByUserId(uid, { fields: { rid: 1 } })
 		.fetch()
-		.map((item) => item.rid);
+		.map((item: ISubscription) => item.rid);
```

Comment:
For consistency, this is not a complete ISubscription object, it's `Pick<ISubscription, 'rid'>`